src/cargo/core/compiler/custom_build.rs

```diff
@@ -268,6 +268,18 @@ fn build_work<'a, 'cfg>(cx: &mut Context<'a, 'cfg>, unit: &Unit<'a>) -> CargoRes
     let prev_output =
         BuildOutput::parse_file(&output_file, &pkg_name, &prev_root_output, &root_output).ok();
     let deps = BuildDeps::new(&output_file, prev_output.as_ref());
+
+    // Check if cargo:rerun-if-changed=<path> is an existing file
+    if !&deps.rerun_if_changed.is_empty() {
+        let path = &deps.rerun_if_changed[0];
```

Comment:
Why just the first one?

Reply:
Ah I didn't consider this since the readme never talks about several files.
How would I do this?
Several entries like this?:
````
rerun-if-changed=file1
rerun-if-changed=file2

Reply:
Yep, like `rebuild_only_on_explicit_paths` tests